lib/tasks/pacbio_data.rake

```diff
@@ -74,10 +74,11 @@ namespace :pacbio_data do
 
     puts '-> Creating pacbio runs...'
     pool_records.each_with_index do |pool, i|
-      run = Pacbio::Run.create!(name: "Run#{pool.id}", sequencing_kit_box_barcode: "SKB#{pool.id}", dna_control_complex_box_barcode: "DCCB#{pool.id}")
-      plate = Pacbio::Plate.create!(run:)
-      Pacbio::Well.create!(plate:, pools: [pool], movie_time: 20, on_plate_loading_concentration: 1,
-                           row: 'A', column: i + 1, generate_hifi: 'In SMRT Link', ccs_analysis_output: 'Yes', binding_kit_box_barcode: "BKB#{pool.id}")
+      run = Pacbio::Run.create(name: "Run#{pool.id}", sequencing_kit_box_barcode: "SKB#{pool.id}", dna_control_complex_box_barcode: "DCCB#{pool.id}")
+      plate = Pacbio::Plate.create(run:)
+
+      well = Pacbio::Well.create(plate:, pools: [pool], movie_time: 20, on_plate_loading_concentration: 1, row: 'A', column: i + 1, generate_hifi: 'In SMRT Link', ccs_analysis_output: 'Yes', binding_kit_box_barcode: "BKB#{pool.id}")
```

Comment:
Ah yes. 
Run and Well can have the `!`. However, Plate does validation to ensure it has wells before it can be saved. Otherwise you get:
`ActiveRecord::RecordInvalid: Validation failed: Wells there must be at least one well`. 

But the `plate.wells = [well]` is not needed either. 

Will clean it up